client/js/test/table-model.test.js

```diff
@@ -1,18 +1,18 @@
 const TableModel = require('../table-model');
 
 describe('table-model', () => {
-	it('can set then get a value', () => {
-		const model = new TableModel();
-		const location = { row: 3, col: 5}
+  it('can set then get a value', () => {
+    const model = new TableModel();
+    const location = { row: 3, col: 5};
 
```

Comment:
Remove extra line here. For tests, you can keep all the lines up to an "expect" statement together. After an expect statement, if you have more tests to run in the same block, use one space.